qa/workunits/fs/misc/dac_override.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh -x
+
+set -e
+sudo mkdir -p testdir
```

Comment:
Wouldn't you want to create this directory as a normal user for the purpose of testing this fix?

So something like
```
mkdir -p testdir

echo "foo" >  testdir/${file}

chmod 600 testdir

# below should fail
cat testdir/${file}

# below should pass as root can override
sudo cat testdir/${file}
```

Reply:
This can be done.  I just used the testcase from the reporter.